Connect4.py

```diff
@@ -1,7 +1,92 @@
 GRID_WIDTH = 7
 GRID_HEIGHT = 6
 
-grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+class Board:
+
+    def __init__(self):
+        self.grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+
+    def draw(self):
+        print("\n  0    1    2    3    4    5    6")
```

Comment:
You can do something like:
"   ".join([str(i) for i in range(GRID_WIDTH)])
or use tabs when printing "\t".

ideally, the game should work if you just update the grid width constant so it's nice if this is dependent on GRID_WIDTH.